rubygem-hammer_cli_foreman_salt/rubygem-hammer_cli_foreman_salt.spec

```diff
@@ -8,8 +8,8 @@
 
 Summary: Foreman Salt-related commands for Hammer CLI
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.0.4
-Release: 4%{?dist}
+Version: 0.0.5
+Release: 1%{?dist}
```

Comment:
Could you add %{?foreman_dist} in here too please?  This'll allow us to rebuild it on 1.10 (for ruby193) without the version colliding in Koji.